docs/COMPATIBILITY.md

```diff
@@ -326,6 +326,12 @@ Note that MS SQL Server and Oracle are not supported as database backends.
 		<td>TBD</td>
 		<td></td>
 	</tr>
+	<tr>
+		<th><a href="https://www.semantic-mediawiki.org/wiki/Semantic_MediaWiki_4.1.3">4.1.3</a></th>
+		<td>7.10.x</td>
+		<td>Supported</td>
+		<td><a href="https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/5611#issuecomment-1962729300">Confirmed to work with ElasticSearch 7.10.2 (MW 1.39)</a></td>
```

Comment:
Bit confusing to list that ElasticSearch works in the OpenSearch column

Reply:
Indeed, my wrong